spec/mailers/referee_mailer_spec.rb

```diff
@@ -27,7 +25,6 @@
 
     describe 'with reference_form feature flag on' do
```

Comment:
The `describe` here can be removed (it's always true)

Reply:
Good catch!

Reply:
Resolved in c7abaa5